### PR TITLE
Doc fixes, minor examples cleanup

### PR DESCRIPTION
**Pull request by avindra**
* Reword the intro to COOKBOOK to get to the point faster
 * Create buckets in ECOSYSTEM for a little bit of organization
 * try to clarify docs around `validateStatus` and `eject` for the interceptor

Minor: update to bootstrap 4 build in examples files

### CODE REVIEW

File: examples/server.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
 var fs = require('fs');
-var url = require('url');
```
Comment (avindra) [inline examples/server.js]: This is just an unused import.

File: examples/server.js [hunk 2]
```diff
@@ -134,4 +133,8 @@ server = http.createServer(function (req, res) {
   }
 });
 
-server.listen(argv.p || 3000);
+const PORT = argv.p || 3000;
+
+server.listen(PORT);
+
+console.log("Examples running on " + PORT);
```
Comment (avindra) [inline examples/server.js]: Had to read the source code to figure this out. Adding a console log for future devs

**Comment by avindra** [comment]
@felipewmartins rebased out the README fixes already merged

can you have a look? it's focused on general local dev stuff, no source changes

**Comment by felipewmartins** [comment]
Hi @avindra 
Awesome! I will merge.

**Pull request merged**